lib/stamp/translator.rb

```diff
@@ -100,7 +99,7 @@ def two_digit_hour_emitter(token)
         Emitters::TwoDigit.new(:hour)
       else
         # 12-hour clock with leading zero
-        Emitters::TwoDigit.new(:hour, &HOUR_TO_12_HOUR)
+        Emitters::TwelveHour.leading_zero
```

Comment:
heh. this is creeping. I remember you once telling me to keep my PRs small.

If you want me to put that into here, just holler, but I'd prefer to create another PR with unit tests and factories for the remaining emitters.